meta-dts-distro/recipes-dts/dts/files/ec_transition

```diff
@@ -65,6 +77,21 @@ board_config() {
           COMPATIBLE_EC_FW_VERSION="2022-08-31_cbff21b"
           EC_HASH="d1001465cea74a550914c14f0c8f901b14827a3b5fa0b612ae6d11594ac2b405  /tmp/ecupdate.rom"
           BIOS_HASH="d4c30660c53bac505997de30b9eac4c5ac15f3212c62366730dc2ca3974bba18  /tmp/biosupdate.rom"
+          PROGRAMATOR_BIOS="ite_ec"
```

Comment:
It sounds like we use `ite_ec` for BIOS flashing, which is not true (I hope).

Also, there is no such word as `programator`. It should be rather: `programmer`

Reply:
@macpijan we used `internal` for BIOS flashing, the variable was not correctly used. https://github.com/Dasharo/meta-dts-ce/pull/1/commits/66ed8e5643d415ba8315bc370b9cf1848c1181c9#diff-6b967df50321ce5b8e4cb1356effadcaf2cc9b4a9a49509c845a030f470d6269L153

Now we named it `PROGRAMMER` and set to use internal. https://github.com/Dasharo/meta-dts-ce/pull/1/commits/66ed8e5643d415ba8315bc370b9cf1848c1181c9#diff-6b967df50321ce5b8e4cb1356effadcaf2cc9b4a9a49509c845a030f470d6269R80